src/josepy/util.py

```diff
@@ -1,31 +1,12 @@
 """JOSE utilities."""
+from typing import Union, Any, Callable, Iterator, Tuple
 from collections.abc import Hashable, Mapping
 
-import OpenSSL
+from OpenSSL import crypto
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric import ec, rsa
 
 
-class abstractclassmethod(classmethod):
```

Comment:
I think this is technically a part of the public API. See https://josepy.readthedocs.io/en/stable/api/util/#josepy.util.abstractclassmethod.

I agree we should deprecate and remove it at some point though.

Reply:
Ok. I reimplemented the decorator `abstractclassmethod` using the standard decorators, and put the usual warning approach for deprecated attributes modules in place.